renderer/components/Pay/Pay.js

```diff
@@ -346,10 +345,19 @@ class Pay extends React.Component {
    * Progress to next form step.
    */
   nextStep = () => {
-    const { currentStep } = this.state
-    const nextStep = Math.min(this.steps().indexOf(currentStep) + 1, this.steps().length - 1)
-    if (currentStep !== nextStep) {
-      this.setState({ currentStep: this.steps()[nextStep], previousStep: currentStep })
+    const { currentStep, invoice } = this.state
+    const nextStepIndex = Math.min(this.steps().indexOf(currentStep) + 1, this.steps().length - 1)
+    const nextStep = this.steps()[nextStepIndex]
+
+    // update route
+    if (invoice && nextStep === 'summary') {
```

Comment:
I feel like this would be better handled in `componentDidUpdate` as it muddies the scope of the `nextStep` method a little.